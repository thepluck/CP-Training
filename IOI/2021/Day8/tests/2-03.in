13 100
4 8
8 11
10 1
3 1
9 6
6 5
4 12
7 4
5 4
4 3
2 1
13 4
100
1 5 62
1 9 47
1 13 18
1 7 2
2 7
2 13
1 7 14
1 2 21
2 9
1 4 77
1 13 0
2 13
2 5
1 12 58
1 8 2
2 2
1 10 20
2 4
2 7
1 13 14
1 5 3
2 8
1 2 5
2 12
2 5
2 13
1 9 46
2 9
1 9 6
1 11 57
2 2
1 8 69
1 5 9
2 9
1 3 80
2 8
2 3
1 6 8
1 3 71
2 11
2 5
1 9 5
1 12 47
2 9
1 11 16
2 6
2 3
2 10
1 5 28
1 3 93
1 4 92
2 12
1 13 26
2 4
1 12 1
2 12
1 12 6
2 11
2 12
1 7 36
1 12 0
2 5
2 13
1 9 11
1 2 6
2 7
1 8 78
2 8
2 2
2 12
1 6 37
1 5 91
2 3
1 12 0
1 11 9
2 12
1 10 0
2 9
2 5
1 2 50
1 5 41
2 11
2 10
1 3 48
1 12 7
2 3
2 12
1 11 5
1 12 1
2 2
2 5
1 4 90
1 5 58
2 4
1 3 95
2 11
2 5
2 6
1 7 76
1 5 13
