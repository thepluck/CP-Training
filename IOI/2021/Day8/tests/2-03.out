654758428
100657091
802386173
49729680
7691200
49729680
802386173
168292800
1920
241920
60480
48384
60480
192648271
2772
630
273000
1050
5148
9324315
5148
1050
4620
10
749743680
33528105
162650733
163011640
162650733
364631176
125970
3124550
1287
112
1540
825
639540
6395400
1046520
22680
850800897
889724051
987747123
746130
6715170
90
64350
3124550
178066716
4976400
234900
62640
239553472
36018000
274833000
36018000
274833000
17852400
34264693
6885161
548100
313200
83692224
363782758
732906720
40717040
780537053
19908
780537053
49666856
558752130
917744023
1008
48620
24310
1
1
1
38
481743814
12540
38
434720
923780
195624
114
1368
923780
12540
3192
675862862
800497943
715913186
39711672
224516325
41783280
598019604
220525380
196402065
10094700
277830
